tests/functional/ComboPopup.js

```diff
@@ -570,6 +690,21 @@ define([
 			return checkMultiSelection(remote, "combo3");
 		},
 
+		"multi selection selection (combo3)": function () {
+			var remote = this.remote;
+
+			if (remote.environmentType.browserName === "internet explorer") {
+				// https://github.com/theintern/leadfoot/issues/17
+				return this.skip("click() doesn't generate mousedown/mouseup, so popup won't open");
+			}
+			if (remote.environmentType.platformName === "iOS") {
+				// https://github.com/theintern/leadfoot/issues/61
+				return this.skip("click() doesn't generate touchstart/touchend, so popup won't open");
+			}
+
+			return checkMultiSelectionCancel(remote, "combo3");
```

Comment:
Since you're only using the `checkMultiSelectionCancel()` function once, maybe you should just inline the code into this test.